packages/terra-i18n/src/i18nLoader.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable */
+import Intl from './intl';
 import loadIntl from './intlLoaders';
 import loadTranslations from './translationsLoaders';
-
 import supportedLocales from './i18nSupportedLocales';
 
 const hasIntl = typeof (Intl) !== 'undefined';
```

Comment:
If we're guaranteed to have Intl defined at this point, we should be able to remove the branching logic around this

Reply:
I had thought about this some, I believe the branching logic is still needed. My understanding of it is that while we do have Intl at this point (either from the native browser implementation or the polyfilled Intl), this check determines if we need to load the polyfilled Intl locale data which is needed if we not using the native browser implementation for Intl)